unit-test/background/reference-tests/referrer-trimming-tests.js

```diff
@@ -91,16 +93,17 @@ for (const setName of Object.keys(testSets)) {
                         enumerable: true,
                         writeable: true
                     })
-                    FakeDocument.prototype.location = {
```

Comment:
Removed this purely as we're not using it, instead we're using `document.URL` inside C-S-S